packages/kbn-failed-test-reporter-cli/failed_tests_reporter/add_messages_to_report.test.ts

```diff
@@ -53,7 +53,7 @@ it('rewrites ftr reports with minimal changes', async () => {
     reportPath: Path.resolve(__dirname, './__fixtures__/ftr_report.xml'),
   });
 
-  expect(createPatch('ftr.xml', FTR_REPORT, xml, { context: 0 })).toMatchInlineSnapshot(`
```

Comment:
I suggest:
```ts
createPatch('ftr.xml', FTR_REPORT, xml, null, null, { context: 0 })
```
And get a review from the code owners

Reply:
I was also thinking about
```js
createPatch('ftr.xml', FTR_REPORT, xml, null, null, { context: 0 })
```
but this would probably change the output, since by default `context` value is 4. 
I am fine with either option if a codeowner okays it @elastic/kibana-operations @elastic/appex-qa

Reply:
I opened https://github.com/elastic/kibana/issues/172569, we'll re-investigate out of band.  The current approach is fine.  Thank you for the fix!

Reply:
@jbudz Actually [it turned out](https://buildkite.com/elastic/kibana-pull-request/builds/181776#018c3a0f-5765-4f17-8727-c29c8a7625e0) that there is a slight difference if you remove `{ context: 0 }`. Without it there's no `[Object object]` in the snapshot, so I went ahead and updated the snapshot.